api/URL.json

```diff
@@ -832,9 +827,15 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": {
-              "version_added": "7.7.0"
-            },
+            "nodejs": [
+              {
+                "version_added": "7.7.0"
+              },
+              {
+                "version_added": "6.13.0",
+                "version_removed": "7.0.0"
+              }
+            ],
```

Comment:
I couldn't find evidence for `toJSON` being available in 6.13.0 in the [release notes](https://nodejs.org/en/blog/release/v6.13.0/). It mentions "WHATWG URL api support", but the corresponding PR doesn't include `toJSON`.

In fact, it was implemented separately in https://github.com/nodejs/node/pull/11236, which wasn't backported to Node v6.

```suggestion
            "nodejs": {
              "version_added": "6.13.0"
            },
```

Reply:
per https://github.com/nodejs/node/issues/55806#issuecomment-2468591380, this is done via https://github.com/nodejs/node/pull/17365

see also https://github.com/nodejs/node/pull/55818

and though the release note doesn't mention it, the test shows it is supported in v6.13.0

![image](https://github.com/user-attachments/assets/8053ba4c-6899-4efe-a200-292b628eeaf9)